cross-chain/arbitrum/hardhat.config.ts

```diff
@@ -0,0 +1,119 @@
+import type { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
+import "@nomiclabs/hardhat-waffle"
+import "@nomiclabs/hardhat-etherscan"
+import "hardhat-gas-reporter"
+import "hardhat-contract-sizer"
+import "hardhat-deploy"
+import "@typechain/hardhat"
+import "hardhat-dependency-compiler"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+    ],
+  },
+
+  paths: {
+    artifacts: "./build",
+  },
+
+  networks: {
+    hardhat: {
+      deploy: ["deploy_l2"],
+    },
+    goerli: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 5,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    mainnet: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 1,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    arbitrumGoerli: {
+      url: process.env.CHAIN_L2_API_URL || "",
+      chainId: 421613,
+      deploy: ["deploy_l2"],
+      accounts: process.env.ACCOUNTS_L2_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L2_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["arbiscan"],
+      companionNetworks: {
+        l1: "goerli",
+      },
```

Comment:
Due to this `companionNetworks` link to `goerli` whenever we execute deployment on `arbitrumGoerli` it requires `CHAIN_L1_API_URL` to be defined.
Since here we don't deploy anything to L1 maybe we could comment out this piece of code?
```suggestion
      // companionNetworks: {
      //   l1: "goerli",
      // },
```

We shouldn't remove it completely as we will use `companionNetworks` eventually in other modules or maybe here as well.

Reply:
ok, commented out